service/controller/v10/resource/s3bucket/resource_test.go

```diff
@@ -2,6 +2,9 @@ package s3bucket
 
 import (
 	"testing"
+
+	awsservice "github.com/giantswarm/aws-operator/service/aws"
+	"github.com/giantswarm/micrologger/microloggertest"
```

Comment:
Nitpick but awsservice should be in a separate block.

Reply:
but is an external project, why should be in different block?

Reply:
The aws service is part of aws-operator so it should be in a separate block.

Reply:
Jesus Christ I was only reading the micrologger one 
Thanks for pointing this out

Reply:
Jeje np :)